### PR TITLE
Tweak docker compose setup

### PR DESCRIPTION
**Pull request by arnested**

### CODE REVIEW

File: drupal8-example/docker-compose.yml [hunk 1]
```diff
@@ -1,36 +1,73 @@
...
-    image: reload/drupal-apache-fpm
+    image: reload/drupal-apache-fpm:latest
     ports:
       - '80'
+      - '443'
     depends_on:
-      - php
+      - fpm
+    links:
+      - fpm
```
Comment (danquah) [inline drupal8-example/docker-compose.yml]: Why do we need a link?

File: drupal8-example/docker-compose.yml [hunk 2]
```diff
@@ -1,36 +1,73 @@
...
       PATH: '/var/www/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
       PHP_SENDMAIL_PATH: /usr/local/bin/mhsendmail --smtp-addr="mailhog:1025"
+      DRUSH_OPTIONS_URI: https://MY-PROJECT.docker
 
   db:
-    image: reload/db-datadir:reloaddk-latest-fulldb
+    image: mariadb:10
     ports:
       - '3306'
+    environment:
```
Comment (danquah) [inline drupal8-example/docker-compose.yml]: If we use datadirs these environment variables won't do anything. At the very least there should be a comment noting this and maybe describe how to switch to a database-dump.

**Review by danquah — CHANGES_REQUESTED** [review]
That link needs clearification

File: drupal8-example/docker-compose.yml (hunk 1 above)
Comment (arnested) [reply drupal8-example/docker-compose.yml]: We don't. It was needed for a workaround I had in my `docker-compose.override.yml` and should stay there when I need the workaround.

File: drupal8-example/docker-compose.yml (hunk 2 above)
Comment (arnested) [reply drupal8-example/docker-compose.yml]: Will fix.

**Comment by arnested** [comment]
How about now, @danquah? (If approved I'll rebase the last to commits into previous commits).

File: drupal8-example/docker-compose.yml [hunk 3]
```diff
@@ -21,16 +21,56 @@ services:
...
       - './:/var/www:rw,cached'
+      - './docker/my_init.d/install-cron.sh:/etc/my_init.d/install-cron.sh'
     environment:
       PATH: '/var/www/vendor/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin'
       PHP_SENDMAIL_PATH: /usr/local/bin/mhsendmail --smtp-addr="mailhog:1025"
+      DRUSH_OPTIONS_URI: https://MY-PROJECT.docker
 
   db:
-    image: reload/db-datadir:reloaddk-latest-fulldb
+    image: mariadb:10
```
Comment (danquah) [inline drupal8-example/docker-compose.yml]: Sorry, missed a minor. The datadir builder uses mariadb:10.3.9 - we might as well keep them in sync. (https://github.com/reload/docker-db-datadir-builder/blob/master/src/build-datadir-image.sh#L145). I mainly lock the version down as I have to make a tweak to the entrypoints script - when https://github.com/docker-library/mysql/pull/471 eventually gets resolved and makes it into mariadb we can probably slack the version constraint a bit.

**Review by danquah — APPROVED** [review]
Looks good! I had an additional comment, but I'm also ok with adding it to a future PR.

File: drupal8-example/docker-compose.yml (hunk 3 above)
Comment (arnested) [reply drupal8-example/docker-compose.yml]: I pushed a commit locking this one down as well.

**Pull request merged**